### ISSUE TITLE
Keyboard user cannot select citation in IE11

### ISSUE THREAD

**Issue by mejackreed** [projectblacklight/blacklight#901]
The keyboard user, using IE 11, was not able to select the citations using either ctrl-A or shift-down-arrow. They both work in Firefox.
